constance/backends/database/apps.py

```diff
@@ -0,0 +1,8 @@
+from django.apps import AppConfig
+from django.utils.translation import ugettext_lazy as _
+
+
+class ConstanceConfig(AppConfig):
+    name = 'constance.backends.database'
+    label = 'constance_db'
```

Comment:
It may be solved this way
https://github.com/omab/python-social-auth/commit/361f42235fe51f422af029f325d041407a16036c

social auth had the same problem with app name "default" (even worse than "database")